cmd/k3s/main.go

```diff
@@ -173,11 +172,11 @@ func internalCLIAction(cmd, dataDir string, args []string) func(ctx *cli.Context
 
 // stageAndRunCLI calls an external binary.
 func stageAndRunCLI(cli *cli.Context, cmd string, dataDir string, args []string) error {
-	return stageAndRun(dataDir, cmd, args)
+	return stageAndRun(dataDir, cmd, args, true)
 }
 
 // stageAndRun does the actual work of setting up and calling an external binary.
-func stageAndRun(dataDir, cmd string, args []string) error {
+func stageAndRun(dataDir, cmd string, args []string, calledAsInternal bool) error {
```

Comment:
Could you briefly explain why you need `calledAsInternal` for windows?

Reply:
for `stageAndRun`, it could be possible to be invoked from different invocation paths (E.g., `k3s.exe kubectl xxxx` by multi-call binary or `kubectl.exe xxxx` by soft linking) and in order to convert the `k3s.exe kubectl xxx` into `kubectl.exe xxx`, I make up a flag to know whether the control path is coming from.

Reply:
I see, thanks